check_patroni/cli.py

```diff
@@ -609,6 +626,13 @@ def node_is_leader(ctx: click.Context, check_standby_leader: bool) -> None:
     default=False,
     help="check if the replica is synchronous",
 )
+@click.option(
+    "--sync-type",
+    type=click.Choice(["any", "sync", "quorum"], case_sensitive=True),
```

Comment:
@dlax in python, is there a point / is it good practice to use an enum here ?

Reply:
Not much here as far as I can tell; but using a `sync_type: Literal["any", "sync", "quorum"]` in signatures below might be beneficial for type checking purpose.